Procfile

```diff
@@ -0,0 +1,3 @@
+# Run Rails & Webpack concurrently
+rails: bundle exec bin/rails server -b 0.0.0.0
```

Comment:
this might default to an env variable, as some people might want to run it with other servers (e.g. puma)

also, there is no need to run bundle exec and rails as it load bundler itself.